explorer/src/providers/accounts/index.tsx

```diff
@@ -226,9 +234,26 @@ async function fetchAccountInfo(
               break;
 
             case "spl-token":
+              const parsed = create(info, TokenAccount);
+              let nftData;
+
+              // Generate a PDA and check for a Metadata Account
+              if (parsed.type === "mint") {
+                const metadata = await getMetadata(pubkey, url);
+                if (metadata) {
+                  // We have a valid Metadata account. Try and pull a Master Edition.
+                  // TODO - Support Editions minted from a Master Edition
```

Comment:
This actually stops us from displaying any information for edition mints. I'll update this PR with a fix. Will try and sneak in a change to show this is an Edition of a MasterEdition.